docs/tutorial/import_export.rst

```diff
@@ -36,11 +36,24 @@ The ``psamm-import`` program supports the import of models in various formats.
 For the SBML format, it supports the COBRA-compliant SBML specifications, the FBC
 specifications, and the basic SBML specifications in levels 1, 2, and 3;
 for the JSON format, it supports the import of JSON files directly from the
-`BiGG`_ database or from locally downloaded versions;
-the support for importing from Excel file is model specific and are available
-for 17 published models. There is also a generic Excel import for models
-produced by the ModelSEED pipeline. To see a list of these models or model
-formats that are supported, use the command:
+`BiGG`_ database or from locally downloaded versions.
+
+The support for importing from Excel file is model specific and are available
+for 17 published models. This import requires the installation of the separate
+psamm-import repository. There is also a generic Excel import for models
+produced by the ModelSEED pipeline.
```

Comment:
Maybe mention that this is for the old ModelSEED? It doesn't work for the new ModelSEED as far as I remember.